examples/for-tests/src/App.js

```diff
@@ -721,6 +721,25 @@ function getFormFields() {
     return formFields;
 }
 
+function getSignInFormFields() {
```

Comment:
we also need to test the reset password screen can have a default value in a similar way